ios/Components/ApplePayModule.swift

```diff
@@ -42,6 +42,40 @@ internal final class ApplePayModule: BaseModule {
         present(component: applePayComponent)
     }
 
+    @objc
+    func isAvailable(_ paymentMethodDict: NSDictionary,
+                     configuration: NSDictionary,
+                     resolver: @escaping RCTPromiseResolveBlock,
+                     rejecter: @escaping RCTPromiseRejectBlock) {
+      let parser = RootConfigurationParser(configuration: configuration)
+      let applePayParser = ApplepayConfigurationParser(configuration: configuration)
+      let paymentMethod: ApplePayPaymentMethod
+      let paymentRequest: PKPaymentRequest
+      guard let payment = parser.payment else {
+        return resolver(false)
+      }
+
+      do {
+        let data = try JSONSerialization.data(withJSONObject: paymentMethodDict, options: [])
+        paymentMethod = try JSONDecoder().decode(ApplePayPaymentMethod.self, from: data)
+        paymentRequest = try applePayParser.buildPaymentRequest(payment: payment)
+      } catch {
+        return resolver(false)
+      }
+
+      let supportedNetworks = paymentMethod.supportedNetworks
+      guard applePayParser.allowOnboarding || PKPaymentAuthorizationViewController.canMakePayments(usingNetworks: supportedNetworks) else {
```

Comment:
Is it possible to test this with `PKPaymentAuthorizationViewController` being a hard dependency?

Reply:
Address in [9e5148a](https://github.com/Adyen/adyen-react-native/pull/701/commits/9e5148a841b1af4fc80b64331eb24eb7b912b8ad)